src/index.html

```diff
@@ -4,9 +4,466 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- <h1>Catalog</h1> -->
+
+
+    <header class="header">
+      <a href="https://www.moyo.ua" class="header__logo">
+        <img src="./images/logo.png" alt="MOYO">
+      </a>
+
+      <nav class="header__nav-bar">
+        <ul class="header__menu">
+          <li class="header__item">
+            <a href="" class="header__link is-active">apple</a>
+          </li>
+          <li class="header__item">
+            <a href="" class="header__link">samsung</a>
+          </li>
+          <li class="header__item">
+            <a href="" class="header__link">смартфоны и телефоны</a>
+          </li>
+          <li class="header__item">
+            <a
+              href=""
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="" class="header__link">гаджеты</a>
+          </li>
+          <li class="header__item">
+            <a href="" class="header__link">планшеты</a>
+          </li>
+          <li class="header__item">
+            <a href="" class="header__link">фото</a>
+          </li>
+          <li class="header__item">
+            <a href="" class="header__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__content">
+
+        <div class="product" data-qa="card">
+          <div class="product__img-box">
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+              class="product__img"
+            >
+          </div>
+
+          <div class="product__info">
+            <p class="product__name">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </p>
+
+            <div class="product__code">
+              Код товара: 195434
+            </div>
+
+            <div class="product__rating rank">
+              <ul class="rank__stars">
+                <li class="rank__star"></li>
+                <li class="rank__star"></li>
+                <li class="rank__star"></li>
+                <li class="rank__star"></li>
+                <li class="rank__star"></li>
+              </ul>
```

Comment:
```suggestion
              <ul class="rank__stars--4">
                <li class="rank__star"></li>
                <li class="rank__star"></li>
                <li class="rank__star"></li>
                <li class="rank__star"></li>
                <li class="rank__star"></li>
              </ul>
```
Передавай модификатор для того чтобы определять сколь звездочек должно быть

Reply:
Думаю это можно делать не списком, а дивами